### ISSUE TITLE
UserWarning: Implicit dimension choice for softmax has been deprecated

### ISSUE THREAD

**Issue by lxt3** [lxt3/bGAN_pytorch#7]
In:
`ComplementCrossEntropyLoss.py:29: UserWarning: Implicit dimension choice for softmax has been deprecated. Change the call to include dim=X as an argument.
  torch.log(1. - torch.nn.functional.softmax(input) + 1e-4)`
